script/generate-emoji-modifiers

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+// Generates the modifiers.js module with a list of emoji that support
+// skin tone modifiers.
+//
+// When new emoji are published in updated Unicode versions:
+//
+// 1. Add the emoji to muan/emojilib and publish a release
+// 2. npm update emojilib
+// 3. npm run generate-emoji
+// 4. git commit -am "Update emoji list"
+
+const emojis = require('emojilib').lib
+
+const pluck = key => obj => obj[key]
+const isFitzpatrick = pluck('fitzpatrick_scale')
+const emoji = pluck('char')
+const codepoints = text => [...text].map(ch => ch.codePointAt(0))
+const hex = num => `0x${num.toString(16)}`
+const first = pluck(0)
+
+const points = new Set(
+  Object.values(emojis)
+    .filter(isFitzpatrick)
+    .map(emoji)
+    .map(codepoints)
+    .map(first)
+    .map(hex)
+)
+
+const script = `
+// Generated by script/generate-emoji-modifiers.
+// You MUST commit any modifications to this file.
+
+/* @flow strict */
+
+const points = new Set([
+  ${[...points].sort().join(',')}
+])
+
+export function isModifiable(emoji: string): boolean {
+  return points.has(emoji.codePointAt(0))
```

Comment:
Ooof e188ccc reminded me that https://blog.emojipedia.org/service-dog-deaf-person-couples-added-to-2019-emoji-list/#aremixedracefamiliesincluded is a thing.

> Unicode already provides a way for vendors to support mixed-race families, should any wish to do so. These are supported on Windows, and a subset where all family members share a skin tone are also supported on Facebook.
>
> **As of Emoji 12.0, no new family combinations are listed by Unicode, potentially due to the number of combinations required when three or four family members are included.**

They are marked as not modifiable in `emojilib`.

https://github.com/muan/emojilib/blob/abd4322e610a0dd4b38fc7f494c49656791e2263/emojis.json#L1736-L1741

If we want to support this, we'd need to check and modify based on each emoji. If we don't (which I am leaning towards), we'd need to check modifiable based on fully composited emoji.